upstairs/src/upstairs.rs

```diff
@@ -1488,28 +1483,20 @@ impl Upstairs {
          * Grab this ID after extent_from_offset: in case of Err we don't
          * want to create a gap in the IDs.
          */
-        let (gw_id, _) = self.guest.guest_work.submit_job(
-            |gw_id| {
-                if write.is_write_unwritten {
-                    cdt::gw__write__unwritten__start!(|| (gw_id.0));
-                } else {
-                    cdt::gw__write__start!(|| (gw_id.0));
-                }
```

Comment:
do we lose anything with these `gw__write__start`s going away? I know there's an `up_to_ds` probe write below that's telling us the same thing, im not sure if they mean two semantically different things or what

Reply:
It's still around, but moved to the beginning of `Downstairs::create_and_enqueue_write_eob` (right after we get the `JobId`)